tests/sentry/silo/test_silo_aware_transaction_patch.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+import pytest
+from django.db import router
+from django.test import override_settings
+
+from sentry.models import Organization, OrganizationMapping
+from sentry.silo import SiloMode
+from sentry.silo.patches.silo_aware_transaction_patch import (
+    MismatchedSiloTransactionError,
+    siloed_atomic,
+)
+from sentry.testutils import TestCase
+
+
+def is_running_in_split_db_mode() -> bool:
+    return bool(os.environ.get("SENTRY_USE_SPLIT_DBS"))
+
+
+class TestSiloAwareTransactionPatchInSingleDbMode(TestCase):
+    def test_routes_to_correct_db_in_control_silo(self):
```

Comment:
```suggestion
    @pytest.mark.skipif(is_running_in_split_db_mode(), reason="only runs in single db mode")
    def test_routes_to_correct_db_in_control_silo(self):
```

You could use a pytest marker to save some lines 🏌️

Reply:
Ahh this is great, thank you!